bin/git-standup

```diff
@@ -240,7 +240,11 @@ if [[ $in_git_repo != 0 ]]; then
     ## Set delimiter to newline for the loop
     IFS=$'\n'
     ## Recursively search for git repositories
-    PROJECT_DIRS=$(find "$INCLUDE_LINKS" . -maxdepth "$MAXDEPTH" -mindepth 0 -name .git)
```

Comment:
Thank you for the catch. Another fix would be to remove the quotes from `$INCLUDE_LINKS`

Reply:
I implemented this way because of the [shellcheck](https://github.com/koalaman/shellcheck) output:

```
➜ shellccheck git-extras/bin/git-standup

In git-extras/bin/git-standup line 244:
        PROJECT_DIRS=$(find . -maxdepth $MAXDEPTH -mindepth 0 -name .git)
                                        ^-------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
        PROJECT_DIRS=$(find . -maxdepth "$MAXDEPTH" -mindepth 0 -name .git)

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

Reply:
I see - yeah that would be a false positive

Reply:
I can change it to your suggestion. It's your decision.
But until now shellcheck like the file. ;)

Reply:
You can use something like `#shellcheck disable=SC2086` to disable the spurious warning

Reply:
thanks!